drivers/counter/timer_dtmr_cmsdk_apb.c

```diff
@@ -87,9 +87,10 @@ static u32_t timer_dtmr_cmsdk_apb_read(struct device *dev)
 	return value;
 }
 
-static int timer_dtmr_cmsdk_apb_set_alarm(struct device *dev,
-					   counter_callback_t callback,
-					   u32_t count, void *user_data)
+static int timer_dtmr_cmsdk_apb_set_wrap_alarm(struct device *dev,
+					       u32_t ticks,
+					       counter_wrap_callback_t callback,
+					       void *user_data)
```

Comment:
note that there is a change in API allowing to set wrap even when counter is not started yet (https://github.com/zephyrproject-rtos/zephyr/blob/6d0a822ebd54059b154458c63bae6216326a344a/include/counter.h#L317). So `if` from line 99 could be removed.

Reply:
Agreed. I will address this in v3